.github/workflows/images.yaml

```diff
@@ -0,0 +1,27 @@
+on:
+  push:
+    branches:
+      - "eaudetcobello/KU-617"
+  schedule:
+    - cron: "0 0 * * *" # Runs every midnight
+
+jobs:
+  publish:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Skopeo
+        run: |
+          cat <<EOF > $HOME/registry.k8s.io-config.yaml
+          registry.k8s.io:
+            images-by-semver:
+              pause: "= 3.7"
+          EOF
+          docker run -v $HOME:$HOME quay.io/skopeo/stable:v1.15 sync \
+            --src yaml \
+            --dest docker \
+            $HOME/registry.k8s.io-config.yaml ghcr.io/canonical \
```

Comment:
where is this file?

Reply:
It's created just above. https://github.com/canonical/k8s-snap/pull/390/files/97066968126db06dc133097a1bffa0472197d239#diff-d0a3d6684c78a148cbf0725d5fe8b5aab6431da05b698a82c9e015516f3020baR17